srtcore/handshake.cpp

```diff
@@ -300,7 +300,7 @@ std::string srt::SrtFlagString(int32_t flags)
 #define LEN(arr) (sizeof (arr)/(sizeof ((arr)[0])))
 
     std::string output;
-    static std::string namera[] = { "TSBPD-snd", "TSBPD-rcv", "haicrypt", "TLPktDrop", "NAKReport", "ReXmitFlag", "StreamAPI" };
+    static std::string namera[] = { "TSBPD-snd", "TSBPD-rcv", "haicrypt", "TLPktDrop", "NAKReport", "ReXmitFlag", "StreamAPI", "FilterCapable" };
```

Comment:
This field wasn't added properly after adding a corresponding flag.

Reply:
Maybe something for 1.5.4. 🤔

Reply:
I was considering it. All other changes don't disturb anyway.

This one of course isn't a crash, at worst it will report this flag as "unknown".